modules/nw-metalLB-basic-upgrade-operator.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-upgrading-operator.adoc
+
+:_content-type: PROCEDURE
+
+[id="upgrading-metallb-operator_{context}"]
+= Upgrading the MetalLB Operator
+
+You can manually upgrade the MetalLB Operator from a version less than {product-version} by using the {product-title} web console. There are a couple of ways to delete Operators from a cluster and these are described in the additional resources section. 
```

Comment:
"There are a couple of ways" sounds a bit approximate. Can we put a reference to teh official doc pages?